badges/js/functions.js

```diff
@@ -1,33 +1,30 @@
-var data;
+let BADGES = [];

```

Comment:
...why capital? we don't use this anywhere else

Reply:
true true, i thought it being capital cause its only setted once at the start and then its just being read, like a semi-constant

Reply:
it kinda makes sense, i understand why, but we literally don't use this anywhere else so probably best not to lol